app/src/main/java/com/codesoom/assignment/handlers/RootHttpHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.codesoom.assignment.handlers;
+
+import com.codesoom.assignment.App.ResultMessage;
+import com.codesoom.assignment.App.HttpStatusCode;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URI;
+
+/**
+ * 루트 도메인으로 들어왔을 경우 실행될 HttpHandler 입니다.
+ * "/"가 아닌 "/tasks" 로 들어오면 {@link TaskHttpHandler} 로 연결됩니다.
+ */
+
+public class RootHttpHandler implements HttpHandler {
```

Comment:
JavaDoc 주석과 클래스 선언 사이의 공백은 삭제해주도록 합니다.

친절한 설명이 좋습니다. 다만 다음과 같이 생략할 수도 있겠습니다.
```suggestion
/**
 * 루트 도메인 리퀘스트를 처리합니다.
 */
public class RootHttpHandler implements HttpHandler {
```
`/tasks`에 대한 설명은 이 클래스의 책임이 아니므로 여기에 적는 것은 좋지 않습니다.
만약 `/tasks`의 이름이 바뀌거나 기능이 삭제된다면 아무 상관이 없는 이 클래스의 주석도 수정해야 하겠죠.
주석에도 커플링이 생기는 것입니다.

Reply:
아 맞네요 ㅠㅠ 감사합니다! 한 클래스는 하나의 책임만 가지게 만들어야 하는데 SRP 를 몸에 새기도록 하겠습니다!